intelmq/bots/parsers/netlab_360/parser.py

```diff
@@ -21,18 +22,28 @@ def parse_line(self, line, report):
 
             if report['feed.url'] in Netlab360ParserBot.DGA_FEED:
                 event.add('source.fqdn', value[1])
-                event.add('time.source', value[3] + ' UTC')
+                # DGA Feed format is
+                # DGA family, Domian, Start and end of valid time(UTC)
+
+                event.add('time.source', value[2] + ' UTC')
```

Comment:
I dont see how year 2030 is a valid date for any practical use? So can you please rationalize that choice?